ee/control/notify_consumer.go

```diff
@@ -0,0 +1,167 @@
+package control
```

Comment:
consider putting the notify portion in its own package, assuming this also the place the desktop menu consumer would go ... maybe?

`/pkg/ee/control/notfiy`
or
`/pkg/ee/control/consumers/notify`

Reply:
I like both of those options! I'll go with consumers/notify.

(FWIW, the menu consumer lives in `ee/desktop/runner` and the bucket consumer lives in `ee/control`, so I don't think there's a single established location for them yet, unless I'm forgetting something from the phase 3 PR.)